docs/installation.rst

```diff
@@ -8,38 +8,37 @@ Installation
 
 .. code-block:: bash
 
-    $ git clone https://github.com/BeyondTheClouds/enos
-
-You should also choose to go with a virtualenv. Please refer to the `virtualenv
-<https://virtualenv.pypa.io/en/stable/>`_ documentation for further information.
+    $ pip install enos
```

Comment:
What I'd like
* enos:master is in sync with kolla-ansible:master
* enos:stable/<version> is in sync with kolla-ansible:stable/<version>
latest enos package is built from the code on the latest stable branch.

In this situation using the reservation.yaml from the enos:master could be an issue.

So, it depends on our branching strategy.